frontend/amundsen_application/static/package.json

```diff
@@ -136,6 +136,7 @@
     "autosize": "^4.0.2",
     "axios": "0.21.1",
     "core-js": "^3.6.5",
+    "d3": "^6.7.0",
```

Comment:
I would recommend you to instead of bringing the whole d3 object, you bring its modules one by one.